### PR TITLE
[15.0][ADD] project_forecast_line_holidays_public

### PR DESCRIPTION
**Pull request by yankinmax**
Depends on:
- [ ] https://github.com/OCA/project/pull/957

### CODE REVIEW

File: project_forecast_line_holidays_public/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (ivantodorovich) [inline project_forecast_line_holidays_public/__manifest__.py]:
```suggestion
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

**Review by ivantodorovich — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Comment by leemannd** [comment]
Hello @yankinmax Thanks for the work. Could you remove the commit dependency in order to merge it?

**Comment by yankinmax** [comment]
hello @leemannd done

**Pull request merged**